src/AspNet.Security.OAuth.DingTalk/DingTalkAuthenticationConstants.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace AspNet.Security.OAuth.DingTalk
```

Comment:
Please add the copyright header to all the C# files.

Reply:
Has been adjusted